esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
Sorry if I missed the context :-) I agree that empty columns don't serve a purpose, but I'd rather the build process remove empty columns rather than make it harder for someone to add chrome 53-specific data later.